README.adoc

```diff
@@ -43,20 +43,20 @@ Learn how to check the health of microservices on Kubernetes by setting up readi
 
 == What you'll learn
 
-You will learn how to create a health check endpoint for your microservices. Then, you 
-will configure {kube} to use this endpoint to keep your microservices running smoothly.
+You will learn how to create health check endpoints for your microservices. Then, you 
+will configure {kube} to use these endpoints to keep your microservices running smoothly.
 
 MicroProfile Health allows services to report their health, and it publishes the overall 
-health status to a defined endpoint. If a service reports `UP`, then it's available. If the service reports `DOWN`, then it's unavailable.
-MicroProfile Health reports an individual service status at 
-the endpoint and indicates the overall status as `UP` if all the services are `UP`. A 
-service orchestrator can then use the health statuses to make decisions.
+health status to a defined endpoint. If a service reports `UP`, then it's available. If 
+the service reports `DOWN`, then it's unavailable. MicroProfile Health reports an individual 
+service status at the endpoint and indicates the overall status as `UP` if all the services 
+are `UP`. A service orchestrator can then use the health statuses to make decisions.
 
 {kube} provides liveness and readiness probes that are used to check the health of your 
-containers, you will work with readiness probes. These probes can check certain files in 
-your containers, check a TCP socket, or make HTTP requests. MicroProfile Health exposes 
-a health endpoint on your microservices. {kube} polls the endpoint as specified by the 
-probes to react appropriately to any change in the microservice's status. Read the 
+containers. These probes can check certain files in your containers, check a TCP socket, 
+or make HTTP requests. MicroProfile Health exposes a readiness and liveness endpoint on 
+your microservices. {kube} polls these endpoints as specified by the probes to react 
+appropriately to any change in the microservice's status. Read the 
 https://openliberty.io/guides/microprofile-health.html[Adding health reports to microservices^] 
```

Comment:
Change
"MicroProfile Health exposes a readiness and liveness endpoint on your microservices."
To
"MicroProfile Health exposes readiness and liveness endpoints on your microservices. "

**Thinking this is correct in this instance.**